### PR TITLE
[Docker]Provide a alpa runtime docker image

### PR DESCRIPTION
**Pull request by wgimperial**
env:cuda113/cudnn820
eg : docker pull ggkrepo/alpa:0.0.1

### CODE REVIEW

File: docker/serving_alpa.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    echo "conda activate base" >> ~/.bashrc
+
+# install conda alpa env
+RUN . /opt/conda/etc/profile.d/conda.sh && \
+    conda create --name alpa python=3.8 -y && \
+    conda activate alpa && \
+    apt install coinor-cbc -y && \
+    pip3 install --upgrade pip && \
+    pip3 install cupy-cuda113 -i https://pypi.douban.com/simple && \
```
Comment (zhisbug) [inline docker/serving_alpa.Dockerfile:27]: remove douban's index?

File: docker/serving_alpa.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    pip3 install alpa  -i https://pypi.douban.com/simple && \
+    pip3 install --trusted-host 169.229.48.123 -i http://169.229.48.123:8080/simple/ jaxlib==0.3.5+cuda113.cudnn820
```
Comment (zhisbug) [inline docker/serving_alpa.Dockerfile:29]: this is deprecated -- we now have a new pypi index

**Review by zhisbug — CHANGES_REQUESTED** [review]
Thanks for the contribution! @wgimperial I left a few comments

File: docker/serving_alpa.Dockerfile (hunk 2 above)
Comment (wgimperial) [reply docker/serving_alpa.Dockerfile]: Thanks for the reminder, I have updated the Dockerfile.

**Comment by JACKHAHA363** [comment]
Hi, did you manage to pass the `alpa.test_install` inside this docker?

**Comment by zhisbug** [comment]
@JACKHAHA363 Our unittest is based on docker. You can take a look here: https://github.com/alpa-projects/alpa/blob/main/docker/unittest.Dockerfile

**Comment by zhisbug** [comment]
@wgimperial  Thanks for the contributions!

**Pull request merged**